utils/remote_control/navigator_joystick_control/nodes/navigator_joystick.py

```diff
@@ -135,9 +139,7 @@ def joy_cb(self, joy):
         # Station hold
         if station_hold == 1 and station_hold != self.last_station_hold_state:
```

Comment:
I think @whispercoros is redoing all of this, so it'll be a lot cleaner. A lot of this code is based on really old stuff.